database/seeders/DatabaseSeeder.php

```diff
@@ -25,7 +25,12 @@ public function run()
         \App\Models\Review::factory(300)->create();
         \App\Models\Celeb::factory(100)->create();
         \App\Models\Watchlist::factory(15)->create();
-        \App\Models\CelebMovie::factory(200)->create();
+        \App\Models\CelebMovie::factory(500)->create();
+        \App\Models\CelebMovie::factory(100)->create([
+            'celeb_id' => 1,
+            'movie_id' => 1,
+            'character_name' => 'Kelsier'
+        ]);
```

Comment:
Why this duplicate?

Reply:
That was just so I could test the pagination views if it got more than 5 pages of titles for a celeb

Reply:
Hmm... probably we can either remove this and extract it into a seeder or change the factory? That we can remove the duplicates?